scripts/form.js

```diff
@@ -0,0 +1,34 @@
+const submitFinal = document.getElementById('buttonFinal');
+const submitButton = document.createElement('button')
+submitButton.setAttribute('type', 'submit');
+submitButton.textContent = 'Submit';
+submitFinal.appendChild(submitButton);
+
+let total = 0;
+
+$(document).ready(function () {
+  $('#buttonFinal').click(function myAdoption() {
+  const form = document.getElementById('buttonFinal');
+  form.addEventListener('submit', function(event) {
+    event.preventDefault();
+    myAdoption();
+    alert("Thank you. The form information has been received.");
+  });
+  });
+  $('#myForm').click(function() {
```

Comment:
It looks like you attached this to the form instead of the submit button, so it executes every time the form is clicked instead of on submission.